src/commands/deployAzurePipeline/configurePipeline.ts

```diff
@@ -0,0 +1,50 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void | undefined> {
+    const deployToAzureExtensionInstalled = await isDeployToAzureExtensionInstalled();
+    if (!deployToAzureExtensionInstalled) {
+        return undefined;
```

Comment:
I don't think there's any need to return the `undefined` value here.

Reply:
@Tatsinnit I remember we put this up on the suggestion to have a proper failure/success scenario. 
Just double checking because thought this value isn't causing any difference to the flow but just conflicting review comments. Trying to minimize the rework.

Reply:
@kanika1894 cool, I guess check the signature of other places please essentially I just did. So please make it consistent I.e. do not return promise undefined from main function hence you might not need to return undefined.